src/fr/animesama/src/eu/kanade/tachiyomi/extension/fr/animesama/AnimeSama.kt

```diff
@@ -100,16 +102,41 @@ class AnimeSama : ParsedHttpSource() {
     }
 
     override fun chapterListRequest(manga: SManga): Request {
```

Comment:
request methods should ideally only return a request, you can make more requests in the parse method